homeassistant/components/sma/manifest.json

```
@@ -3,7 +3,7 @@
   "name": "SMA Solar",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/sma",
-  "requirements": ["pysma==0.4.3"],
-  "codeowners": ["@kellerza", "@rklomp"],
+  "requirements": ["pysma==0.5.0"],
+  "codeowners": ["@rklomp", "@kellerza"],
```

```suggestion
  "codeowners": ["@kellerza", "@rklomp"],
```
This change is not relevant for the version bump, and there are no priority in CODEOWNERS, when a new person becomes CODEOWNERS, his/hers name is appended.

---

Removed this change from this pull request